control/narrative_table.go

```diff
@@ -0,0 +1,91 @@
+package control
+
+import (
+	"errors"
+	"regexp"
+
+	"github.com/jbowtie/gokogiri/xml"
+	"github.com/opencontrol/fedramp-templater/docx/helper"
+	"github.com/opencontrol/fedramp-templater/opencontrols"
+)
+
+func findSectionKey(row xml.Node) (section string, err error) {
+	re := regexp.MustCompile(`Part ([a-z])`)
```

Comment:
> This is something that the yaml doesn't take care of yet.

--> https://github.com/opencontrol/fedramp-templater/issues/31

> what would they do if the Part ever went beyond `z`

Yeeeeah...let's 🏈 🏃 on that.